static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx

```diff
@@ -105,16 +107,41 @@ function Event({
   // We just need to stack up to 3 times
   const totalStackNumber = Math.min(crumbs.length, 3);
 
+  // If there is only 1 event use the tab navigation handler on the node
+  const nodeClickHandler = () => {
+    if (crumbs.length === 1) {
+      const crumb = crumbs[0];
+
+      switch (crumb.type) {
+        case 'navigation':
+        case 'debug':
+          setActiveTab('network');
+          break;
+        case 'ui':
+          setActiveTab('dom');
+          break;
+        case 'error':
+          setActiveTab('issues');
+          break;
+        default:
+          setActiveTab('console');
+          break;
+      }
+    }
+  };
+
   return (
-    <IconPosition>
+    <IconPosition onClick={nodeClickHandler}>
       <IconNodeTooltip title={title} overlayStyle={overlayStyle} isHoverable>
-        {crumbs.slice(0, totalStackNumber).map((crumb, index) => (
-          <IconNode
-            color={colors[index] || crumb.color}
-            key={crumb.id}
-            stack={{totalStackNumber, index}}
-          />
-        ))}
+        {crumbs.slice(0, totalStackNumber).map((crumb, index) => {
+          return (
+            <IconNode
+              color={colors[index] || crumb.color}
+              key={crumb.id}
+              stack={{totalStackNumber, index}}
+            />
+          );
+        })}
```

Comment:
formatting changes made git history harder to read
```suggestion
        {crumbs.slice(0, totalStackNumber).map((crumb, index) => (
          <IconNode
            color={colors[index] || crumb.color}
            key={crumb.id}
            stack={{totalStackNumber, index}}
          />
        ))}
```